verification/dependencies/net/udpsock.gobra

```diff
@@ -139,4 +140,13 @@ preserves acc(c.Mem(), _)
 preserves acc(sl.Bytes(b, 0, len(b)), R15)
 ensures   err == nil ==> 0 <= n && n <= len(b)
 ensures   err != nil ==> err.ErrorMem()
-func (c *UDPConn) Write(b []byte) (n int, err error)
\ No newline at end of file
+func (c *UDPConn) Write(b []byte) (n int, err error)
+
+ensures acc(u.Mem())
+decreases _
+func UDPAddrFromAddrPort(addr netip.AddrPort) (u *UDPAddr)
+
+ensures   err == nil ==> 0 <= n && n <= len(b)
+ensures   err != nil ==> err.ErrorMem()
```

Comment:
I am actualy not sure if this holds. WriteToUDPAddrPort uses errors like `syscall.EINVAL` and I am not sure if `err.ErrorMem()` holds then.

Reply:
`err.ErrorMem()` means that we should be able to read whatever result is produced. As such, it is fine for us to make this assumption, given that the error value would be totally useless otherwise

Reply:
I see. Thank you.